src/App.js

```diff
@@ -40,21 +41,34 @@ class App extends Component {
   }
 
   filtriraj = filteri => {
+    const imaEngleski = this.state.engleski ? 'en' : 'tekst'
```

Comment:
imaEngleski je zavodljiv naziv, to je zapravo atribut koji moze biti 'en' ili 'tekst'. Iz naziva zvuci kao da je bul.